src/lib/users/proposal.rb

```diff
@@ -102,69 +104,171 @@ def description
 
   private
 
+    attr_reader :users_event_id, :root_event_id
+
+    # The config holding users and groups to create
+    #
+    # @return [Y2Users::Config]
+    def config
+      return @config if @config
+
+      # FIXME: should AutoYaST use `target` configuration as well?
```

Comment:
ideally yes and at the end always merge target and system based on strategy for given mode.

Reply:
Adapted